mailpoet/lib/Util/Notices/PendingApprovalNotice.php

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Util\Notices;
+
+use MailPoet\Mailer\Mailer;
+use MailPoet\Settings\SettingsController;
+use MailPoet\Util\Helpers;
+use MailPoet\WP\Notice as WPNotice;
+
+class PendingApprovalNotice {
+
+  const OPTION_NAME = 'dismissed-pending-approval-notice';
```

Comment:
I'm a little confused about the naming here. This is called an option name, and it has the word `dismissed` in it, but it's not used as an option and it doesn't seem to be dismissable like the others listed in `ajaxDismissNoticeHandler`.